ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -4186,3 +4186,76 @@ presubmits:
           requests:
             cpu: 500m
     trigger: (?m)^/test yaml-lint
+  - agent: kubernetes
+    always_run: true
+    branches:
+      - ^master$
+      - ^master-
+    cluster: build01
+    context: ci/prow/e2e-nutanix
+    decorate: true
+    labels:
+      ci-operator.openshift.io/cloud: nutanix
+      ci-operator.openshift.io/cloud-cluster-profile: nutanix
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-installer-e2e-nutanix
+    rerun_command: /test e2e-nutanix
+    skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|images)/(alibabacloud|aws|baremetal|gcp|ibmcloud|libvirt|openstack|ovirt|vsphere|azure)/)
+    spec:
+      containers:
+        - args:
+            - --gcs-upload-secret=/secrets/gcs/service-account.json
+            - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+            - --lease-server-credentials-file=/etc/boskos/credentials
+            - --report-credentials-file=/etc/report/credentials
+            - --secret-dir=/secrets/ci-pull-credentials
+            - --secret-dir=/usr/local/e2e-nutanix-cluster-profile
+            - --target=e2e-nutanix
+          command:
+            - ci-operator
+          image: ci-operator:latest
+          imagePullPolicy: Always
+          name: ""
+          resources:
+            requests:
+              cpu: 10m
+          volumeMounts:
+            - mountPath: /etc/boskos
+              name: boskos
+              readOnly: true
+            - mountPath: /secrets/ci-pull-credentials
+              name: ci-pull-credentials
+              readOnly: true
+            - mountPath: /usr/local/e2e-nutanix-cluster-profile
+              name: cluster-profile
+            - mountPath: /secrets/gcs
+              name: gcs-credentials
+              readOnly: true
+            - mountPath: /etc/pull-secret
+              name: pull-secret
+              readOnly: true
+            - mountPath: /etc/report
+              name: result-aggregator
+              readOnly: true
+      serviceAccountName: ci-operator
+      volumes:
+        - name: boskos
+          secret:
+            items:
+              - key: credentials
+                path: credentials
+            secretName: boskos-credentials
+        - name: ci-pull-credentials
+          secret:
+            secretName: ci-pull-credentials
+        - name: cluster-profile
+          secret:
+            secretName: cluster-secrets-nutanix
+        - name: pull-secret
+          secret:
+            secretName: registry-pull-credentials
+        - name: result-aggregator
+          secret:
+            secretName: result-aggregator
+    trigger: (?m)^/test( | .* )e2e-aws,?($|\s.*)
```

Comment:
Was this section copied? There is a make target to generate the file.

Reply:
Yes, it was copied. That time we have no access to registry with images required for `make jobs`